test/test_config.ini

```diff
@@ -0,0 +1,10 @@
+# Config File for Virtual JukeBox Testing
+
+[MainParams]
+programName=JukeBox
+IP=192.168.0.101
+port=4711
+
+[SomeMoreParams]
```

Comment:
i guess we should remove the dummy data by now, shouldnt we?

Reply:
this is just the test source, i think this could stay for future test cases

Reply:
yeah didnt see that it is in test